targetconfig.json

```diff
@@ -368,5 +368,117 @@
     },
     "electronManifest": {
         "latest": "v1.12.35"
+    },
+    "kiosk": {
+        "games": [
+            {
+            "id": "_Y8VUmcHYsdsy",
+            "name": "GalgaMulti",
+            "description": "Controlling a starship, both players will destroy the Galga forces, while avoiding enemies. Each player has 3 lives, game ends once a player runs out of lives.",
+            "highScoreMode": "SingleAscending"
```

Comment:
nit: prefer indenting but that's only for readability / doesn't really matter if you don't feel like doing it